charts/influxdb2/README.md

```diff
@@ -1,56 +1,50 @@
-# InfluxDB
+# InfluxDB v2 Helm chart
```

Comment:
In our other Helm charts, we include a configuration options...were they intentionally left out from this chart's readme?